### PR TITLE
getdns 0.2.0

### PR DESCRIPTION
**Pull request by ngoyal**

### CODE REVIEW

File: Library/Formula/getdns.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 class Getdns < Formula
   homepage "http://getdnsapi.net"
```
Comment (vszakats) [inline Library/Formula/getdns.rb]: Could you please change homepage too to https?:
https://getdnsapi.net

Comment (vszakats) [inline Library/Formula/getdns.rb]: Thanks!

File: Library/Formula/getdns.rb [hunk 2]
```diff
@@ -21,7 +22,10 @@ class Getdns < Formula
   depends_on "libev" => :optional
 
   def install
-    args = []
+    args = [
+      "--with-ssl=#{Formula["openssl"].opt_prefix}",
+      "--with-trust-anchor=/usr/local/etc/getdns-root.key"
```
Comment (DomT4) [inline Library/Formula/getdns.rb]: Don't hardcode this to `/usr/local`. Should be `#{etc}/getdns-root.key`.

Comment (ngoyal) [inline Library/Formula/getdns.rb]: Thanks - it's good now - somehow I missed the docs @ https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md#variables-for-directory-locations

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/1879f33d41ea82813c3a739c149fe886726bcd48. Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:.

Couple of small nits for future: The preferred checksum is now `sha256` and commits should be squashed. Otherwise :100:, Thanks!

**Pull request closed without merging**